wbooks/App.js

```diff
@@ -16,19 +16,19 @@ import {
 } from 'react-native';
 
 import BookList from './src/app/screens/booklist';
-import styles from './styles';
+import {createAppContainer} from 'react-navigation';
+import {createStackNavigator} from 'react-navigation-stack';
```

Comment:
we separate imports like this:
```
#Libraries imports
Example: import React from 'react'

#Local imports
Example: import styles from './styles'
```

they are separated by a new line

Reply:
done